web/components/home/Home.js

```diff
@@ -176,17 +176,23 @@ class Home extends React.Component {
                         className="scale-width-mobile"
                       />
                     </Link>
-                    {/** <a
-                      target="_blank"
-                      href="https://docs.google.com/forms/d/1HQDZ2YWv8NCkSznOf-Bp6b14gRENiqoSCWYQ8_rYFLM/edit?usp=drivesdk"
-                      rel="noopener noreferrer"
-                      className="scale-width-mobile"
+                    <Link to="/application/volunteer" className="scale-width-mobile margin-right-s scale-margin-sides-mobile-none scale-margin-bottom-mobile-s">
```

Comment:
Almost perfect, but if you resize your screen width from `480px` to `555px` using Chrome Dev Tools, you'll notice that the button text overflows:
![image](https://user-images.githubusercontent.com/12876696/48745285-acad9b00-ec1f-11e8-90ed-e94ff9527f24.png)


One solution to this is to go to the mobile view (where there's one button per row) sooner. You can do this by making all of the classes between L172 - L196 use `phablet` instead of `mobile`. Specifically:

1. `scale-row-mobile` -> `scale-row-phablet`
2. `scale-width-mobile` -> `scale-width-phablet`
3. `scale-margin-sides-mobile-none` -> `scale-margin-sides-phablet-none`
4. `scale-margin-bottom-mobile-s` -> `scale-margin-bottom-phablet-s`

However I think only `scale-width-phablet` is defined, so you'll have to define the other three. You can find all of the classes in [`web\styles\breakpoints.sass`](https://github.com/nwplus/nwhacks2019/blob/christine/homepage/mentor-volunteer-buttons/web/styles/breakpoints.sass)

https://github.com/nwplus/nwhacks2019/blob/44ea4ad853a1e12e859dafc8bc3ebfdffdfb673d/web/components/home/Home.js#L172-L196

Let me know if you have any questions / need help reproducing the issue!